jekyll/_cci2/github-integration.adoc

```diff
@@ -308,18 +308,41 @@ After access is granted, CircleCI should behave normally again.
 If you would like to rename your organization or repository, follow the xref:rename-organizations-and-repositories#[Rename organizations and repositories] guide to make sure you do not lose access to environment variables or contexts in the process.
 
 [#using-github-app-functionality]
-== Using GitHub App functionality alongside the GitHub OAuth app
+== Using GitHub App functionality alongside the GitHub OAuth App
 
-xref:github-apps-integration#[CircleCI’s GitHub App integration] offers increased control over which resources in your VCS CircleCI can access, and lets CircleCI access those resources in a more secure manner. Specifically:
+Some existing and future functionality on CircleCI can only be enabled through the more secure xref:github-apps-integration#[CircleCI’s GitHub App integration].
+
+Until October 2024, the CircleCI GitHub App Integration was available exclusively to organizations created after September 2023. **Now, all organizations can install the CircleCI GitHub App to access new functionality. This includes organizations that currently integrate with CircleCI's GitHub OAuth App.**
```

Comment:
can BB Cloud orgs do this too?

Reply:
Sorry I'd missed this one! 
They can, but 
1. what's the point, if they use BB? 
2. I figured that in this page - which is specific to the  GitHub oAuth - it would be simpler to remain focused on integrations with GitHub only. 

But I'm happy to approach this differently if you think that's better

Reply:
As long as they _can_ regardless of why I think this is fine. Mainly just thinking the AI bot will pick this up so the range of the content is outside just the GitHub integration page, if that makes sense